decode.go

```diff
@@ -33,6 +34,24 @@ var (
 	}
 )
 
+type Bbox struct {
```

Comment:
I noticed the last commit changed the name of `Header.Bbox` to `Header.BoundingBox` but it didn't include a change of `type Bbox struct` to `type BoundingBox struct`.